src/algo/CircularlyLinkedList.js

```diff
@@ -336,6 +342,13 @@ export default class CircularlyLinkedList extends Algorithm {
 	reset() {
 		this.size = 0;
 		this.nextIndex = this.resetIndex;
+		this.infoLabelID = this.nextIndex++;
```

Comment:
```suggestion
```
Since in setup() you instantiate infoLabelID and create the label prior to when this.resetIndex gets set, you actually don't have to reset infoLabelID here. Every ID assigned prior to line 335 is not affected when the animation is reset.